### PR TITLE
Add monitor meterics for secondary storage

### PR DESCRIPTION
**Pull request by Napoleon0621**
1. add storage add job and delete jobmeterics.
2. fix format

### CODE REVIEW

File: storage/manager.go [hunk 1]
```diff
@@ -109,8 +111,12 @@ func (m *Manager) AddPool(name string, pool engine.Engine, threshold int64) {
 	go pumper.Loop(m.PumpFn(name, pool, threshold))
 }
 
-func (m *Manager) AddJob(ctx context.Context, job *model.JobData) error {
-	return m.storage.BatchAddJobs(ctx, []*model.JobData{job})
+func (m *Manager) AddJob(ctx context.Context, name string, job *model.JobData) error {
+	err := m.storage.BatchAddJobs(ctx, []*model.JobData{job})
+	if err == nil {
+		metrics.storageAddJobs.WithLabelValues(name).Inc()
```
Comment (git-hulk) [inline storage/manager.go:117]: I think we should use namespace/queue instead of the pool name

Comment (Napoleon0621) [reply storage/manager.go]: Yes, just updated. Please have another look.

File: engine/redis_v2/engine.go [hunk 2]
```diff
@@ -90,7 +90,7 @@ func (e *Engine) sink2SecondStorage(ctx context.Context, job engine.Job) error {
 		Tries:       int64(job.Tries()),
 		CreatedTime: now,
 	}
-	return storage.Get().AddJob(ctx, dbJob)
+	return storage.Get().AddJob(ctx, e.redis.Name, dbJob)
```
Comment (git-hulk) [inline engine/redis_v2/engine.go:93]: dbJob has pool name, so don't add it again

File: storage/manager.go [hunk 3]
```diff
@@ -128,8 +130,12 @@ func (m *Manager) AddPool(name string, pool engine.Engine, threshold int64) {
 	}()
 }
 
-func (m *Manager) AddJob(ctx context.Context, job *model.JobData) error {
-	return m.storage.BatchAddJobs(ctx, []*model.JobData{job})
+func (m *Manager) AddJob(ctx context.Context, name string, job *model.JobData) error {
+	err := m.storage.BatchAddJobs(ctx, []*model.JobData{job})
+	if err == nil {
+		metrics.storageAddJobs.WithLabelValues(name, job.Namespace, job.Queue).Inc()
+	}
```
Comment (git-hulk) [inline storage/manager.go:137]: Need to add a label to record add jobs success or failed

**Review by git-hulk — COMMENTED** [review]
LGTM, thanks

**Review by git-hulk — APPROVED** [review]

File: engine/redis_v2/engine.go (hunk 2 above)
Comment (Napoleon0621) [reply engine/redis_v2/engine.go]: > LGTM, thanks

Thanks for the review.
Gonna merge this PR.

**Pull request merged**